metron-platform/metron-data-management/README.md

```diff
@@ -364,7 +364,7 @@ The parameters for the utility are as follows:
 | Short Code | Long Code           | Is Required? | Description                                                                                      |
 |------------|---------------------|--------------|--------------------------------------------------------------------------------------------------|
 | -h         |                     | No           | Generate the help screen/set of options                                                          |
-| -g         | --geo_url           | No           | GeoIP URL - defaults to http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz |
+| -g         | --geo_url           | No           | GeoIP URL - defaults to http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz |
```

Comment:
Are you missing the new -ra option?

Reply:
Actually, this section is dedicated to GeoLite2. Should we also have the ASN details?

Reply:
Added in the `-ra` option as well as `-re` which apparently got added at some point.

ASN is one of the GeoLite2 databases (the other one we have available is actually the City database), so it seems reasonable to keep it together.

Should I rename `AsnDatabase` to `GeoLiteAsnDatabase` to make this more clear (and match `GeoLiteCityDatabase`)?

Reply:
Probably not a bad idea - it would make it clearer